### ISSUE TITLE
Finalize Test Pilot experiment on website

### ISSUE THREAD

**Issue by sandysage** [mozilla-lockbox/mozilla-lockbox.github.io#44]
**Work on the `testpilot` repo is happening here: https://github.com/mozilla/testpilot/pull/3624**

- [x] content & design for the detail page - Requires #47
- [x] create PR for initial content (set to "dev: true") https://github.com/mozilla/testpilot/pull/3624
- [x] add contributors
- [x] set up Discourse thread and link it
  - https://discourse.mozilla.org/c/test-pilot/lockbox
- [x] make sure the store download button is hooked up and looks right
  - [x] make sure "send to SMS/email" is hooked up
- [x] follow-up PR to go live - Requires https://github.com/mozilla-lockbox/mozilla-lockbox.github.io/issues/67
- [x] follow-up PR for final content - Requires https://github.com/mozilla-lockbox/mozilla-lockbox.github.io/issues/66
- [x] make sure "Your privacy" section is sane and approved - https://github.com/mozilla/testpilot/issues/3646
- [x] video for featured experiment - Requires #48

**Comment by sandysage** [comment]
This task is to make sure we've delivered the assets to Test Pilot. Other tickets have been opened to track the work to create the featured experiment and the experiment details.

**Comment by devinreams** [comment]
Cool. Then I can help with all this work once the final content/stuff is ready...

**Comment by devinreams** [comment]
In progress here with follow-up todos for launch:
https://github.com/mozilla/testpilot/pull/3624

**Comment by devinreams** [comment]
>make sure "Your privacy" section makes sense

Long story short, simplest path is to include two bullets that capture the essence and make sure the link points to our external site:

![screen shot 2018-06-25 at 11 33 47 am](https://user-images.githubusercontent.com/49511/41865917-d7e430ea-786b-11e8-97a5-f780eff8a3d6.png)

@sandysage will double check with content/legal teams for final 👍

**Comment by devinreams** [comment]
The above approach to "Your privacy" was revisited. We have 3 bullets as a fallback, but the desired approach is described here (just an intro paragraph with link to the detailed policy): https://github.com/mozilla/testpilot/issues/3646

**Comment by devinreams** [comment]
Updated "Your privacy" section will be like this:

![screen shot 2018-06-27 at 10 27 10 am](https://user-images.githubusercontent.com/49511/41986873-c1110f5c-79f4-11e8-8c82-6d21303394cf.png)

Marking complete ✅

**Comment by devinreams** [comment]
All checkboxes are checked!

All that's left is actually merging the testpilot [PR](https://github.com/mozilla/testpilot/issues/3646) and that's happening later tomorrow (Thursday) or Friday. I'll close this when that's done...

**Comment by devinreams** [comment]
OK, the page content is merged and ready to go live with the next testpilot code deploy. 🚀
